### ISSUE TITLE
How to verify the accuracy of the reference model?

### ISSUE THREAD

**Issue by zhangshushu15** [yjxiong/caffe#106]
Hi,

I'm trying to verify the accuracy reported at

http://personal.ie.cuhk.edu.hk/~xy012/others/action_recog/

for the reference models found at

https://github.com/yjxiong/caffe/tree/action_recog/models/action_recognition

Specifically, I'm looking at the **temporal** model of **split1**. The command I used is:

build/tools/caffe test -model models/action_recognition/vgg_16_flow_train_val_fast.prototxt  -weights cuhk_action_temporal_vgg_16_split1.caffemodel -iterations 100000 -gpu 0

However, the accuracy I got is around 72%, which is 13.7% lower than the reported number. Did I do anything wrong? If yes, what's the recommended way to verify the accuracy?

Thank you.

Best,
Shu

**Comment by yjxiong** [comment]
What you got with Caffe is single crop single frame accuracy. The accuracy reported in the paper is video-level accuracy using 10 crops per frame.
We have provided the scripts to get the video-level prediction in this repo.

**Comment by zhangshushu15** [comment]
Make sense. Let me try again.

Thanks.
